src/Analyzers/CSharp/Tests/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -2641,5 +2710,27 @@ public async Task UsedPrivateExtensionMethod()
 
             await VerifyCS.VerifyCodeFixAsync(code, code);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
+        [WorkItem(48206, "https://github.com/dotnet/roslyn/issues/48206")]
+        public async Task DynamicDependencySimpleCase()
+        {
+            var code = @"
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+
+class Program
+{
+    static void Main() => new Program().Bar();
+
+    [DynamicDependency(""Foo"")]
```

Comment:
is it legal to just name 'Foo' without things like `Foo()`, that seems problematic as we'll have to have some logic to figure out how to map these sorts of things to overload groups.

Reply:
is there a runtime spec for this anywhere that we can look at?

Reply:
@CyrusNajmabadi I'm not sure if there is a spec for this somewhere. But basically writing just the name means **all overloads**.

There are examples for pointing to a **specific** attribute in @stephentoub's comment in the linked issue.